### ISSUE TITLE
Bar numbers in examples

### ISSUE THREAD

**Issue by rettinghaus** [pfefferniels/probstuecke-digital#81]
What to do with measure numbers in examples? If they were just extracts from the whole piece we could leave the original numbers in there. But usually they're more than that. Do they deserve a local independent numbering or should we just drop them entirely? 
For pointing to measures in the score we use `@corresp`. From a technical perspective way to go. But it is not for humans. Should we give a `@label` with the corresponding measure number?

**Comment by pfefferniels** [comment]
Good point – having both, `@corresp` and a `@label` could be quite useful. But I don't see a good reason for a local numbering at the moment … from my side we can drop them.

**Comment by pfefferniels** [comment]
In some point we might want to think about using this linking via `@corresp` to add the examples to the score, instead of keeping a duplicate embedded in the score and removing it every time by default.

This would probably require some heavy XSL magic – I'm not sure if I'm capable of it …
